docs/surveys/developer-survey/README.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Gradle Build Tool - 2024 Software Developer Survey"
+description: >
+  We're doing a developer survey to prioritize our developer experience work.
+  Take the survey and get gift cards and special swag!
+---
+
+# Gradle Build Tool - Software Developer Survey
+
+The Gradle team is committed to enhancing developer experience,
+it is one of the top priorities on [Gradle Roadmap](../..roadmap/README.md/).
+Your feedback is crucial in helping us prioritize our efforts!
+
+<a class="button button--blue"
+   href="https://www.surveymonkey.com/r/7FZPXQZ"
+   target="_blank">Take the Survey
+</a>
+
+## 2024 Developer Survey
+
+![Test](./images/hero.png)
+
+The 2024 survey focuses on Gradle Build Tool developer experience, intends to get feedback on complexity of particular operations and to prioritize them.
+This is why the form is quite long, and we will appreciate your patience.
+The survey will be open for responses until the end of the year,
+and we intend to make it a recurrent event.
+
+<a class="button button--blue"
+   href="https://www.surveymonkey.com/r/7FZPXQZ"
+   target="_blank">Take the Survey
+</a>
+
+## Swag
+
+We offer a gift card raffle for anonymous responses.
+Several most active participants, who provide substantial feedback and have a follow-up with us,
+will get special edition Gradle swag.
+To participate in the raffle,
+submit your feedback by October 31st, 2024.
+
+## Previous Surveys
+
+It is our first one!
```

Comment:
🤔 This seems odd, I'd consider dropping it.

Reply:
I was about verify the survey history and link them if I find something